app/Http/Validation/Validation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Validation;
+
+abstract class Validation implements ValidationInterface
```

Comment:
For this there is already a Laravel construct afaik. This whole "review" endpoint specific validation should live within the request specialisation class imo.
https://laravel.com/docs/9.x/validation#main-content

Reply:
Good point. Removed